sdk/storage/Azure.Storage.Common/swagger/Generator/src/generator.ts

```diff
@@ -617,6 +626,17 @@ function generateOperation(w: IndentWriter, serviceModel: IServiceModel, group:
                     });
                 }
             }
+            if (response.struct) {
+                w.line();
+                const Separator = IndentWriter.createFenceposter();
```

Comment:
nit - don't we case locals as `separator` in TypeScript?

Reply:
Yes, I got confused between Typescript and .Net. Will update it.